services/core/SQLHistorian/README.rst

```diff
@@ -0,0 +1,109 @@
+.. _SQL-Historian:
+
+============
+SQLHistorian
+============
+
+This is a historian agent that writes data to a SQLite or Mysql database
+based on the connection parameters in the configuration. The sql historian has
+been programmed to allow for inconsistent network connectivity
+(automatic re-connection to tcp based databases). All additions to the
+historian are batched and wrapped within a transaction with commit and
+rollback functions properly implemented. This allows the maximum
+throughput of data with the most protection.
+
+MySQL
+~~~~~
+
+Installation notes
+------------------
+
+1. In order to support timestamp with microseconds you need at least
+   MySql 5.6.4. Please see this `MySql documentation
+   <http://dev.mysql.com/doc/refman/5.6/en/fractional-seconds.html>`__
+   for more details
+
+2. The mysql user must have READ, WRITE, UPDATE, and DELETE privileges.
+
+3. The tables in the sql database be created as part of the execution of
+   the SQLHistorianAgent only if the database user has CREATE privileges.
+   If not, use the mysql-create*.sql script to create the tables and then
+   start agent.
+
+Dependencies
+------------
+
+In order to use mysql one must install the **mysql-python connector**
+
```

Comment:
Lets get rid of thee first one as it never works and just go with the
pip install mysql-connector-python-rf

Reply:
fixed